index.html

```diff
@@ -124,5 +124,32 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-section">
+    <div id="join-hiring">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <form action="#">
+        <button type="submit" id="roles-button">See the roles</button>
+      </form>
+    </div>
+
+    <div id="picture-join">
+      <img id="flying-shoe" src="images/joinbox-flying-shoe.png" alt="flying shoe">
+      <img id="pencil-small" src="images/joinbox-pencil-small.png" alt="pencil small">
+      <img id="green-box" src="images/joinbox-abstract-green-box.png" alt="abstract green box">
+      <img id="blue-ball" src="images/joinbox-abstract-blue-ball.png" alt="abstract blue ball">
+      <img id="flying-febby" src="images/joinbox-flying-febby.png" alt="flying febby">
+    </div>
+
+    <div id="earlyacess-hiring">
```

Comment:
Minor typo. earlya*c*cess-hiring. Put a hyphen or something between words to make it more readable.